README.md

```diff
@@ -18,40 +18,58 @@ Tested with a standard 3rd party `oauth2_provider` Django application from
 
 Supported Versions
 ------------------
-Python 2.7 and Django 1.11,
-Python 3.7 and Django 2.2
+- Python 2.7 and Django 1.11
+- Python 3.7 and Django 2.2
+- PostgreSQL >= 9.6.9
+
+Other versions may likely work too, but are not supported.
 
 Python and Django compatibility is maintained in `compat.py` and `test_compat.py`
 
-Quick Start
------------
+### Installation
+-------------
+#### Extras
+As of version `0.2.0` this project provides two extras, `SF` and `providerApp`:
+- `SF` - enables Salesforce JWT grant type support
```

Comment:
I'm doubt that `SF` is a good extras name as JWT grant type is a standard and can be used to connect to the other services in the future.
Please avoid using SF and Salesforce.
Could you also add a link to the RFC https://tools.ietf.org/html/rfc7519 ?

Reply:
all good points, done